### ISSUE TITLE
Date Output Serialization - has time component added

### ISSUE THREAD

**Issue by bgribaudo** [microsoft/vscode-powerquery-sdk#100]
When a `date` is output, a time component is added to its serialized rendering. In the output, this makes it impossible to differentiate between a `datetime` with hour, minute and second set to zero, and a plain `date`.

It would be nice to be able to clearly differentiate between `date` and `datetime` values in the output. 

Say I'm using a CI/CD tool to run a test PQ script, then have the tool compare the produced output with a set expected values. The inability to discern between some `date` and `datetime` values could lead to incorrect test results.

## Example of Current Behavior
````
[DataSource.Kind="BenTest", Publish="BenTest.Publish"]
shared BenTest.Contents = () =>
    #table(
        null, 
        {
            {#date(2022, 1, 1)},
            {#datetime(2022, 1, 1, 0, 0, 0)}
        }
    );
````

Outputs as follows: 
````
"Output": [
    {
        "Column1": "2022-01-01T00:00:00"
    },
    {
        "Column1": "2022-01-01T00:00:00"
    }
]
````

**Comment by bgribaudo** [comment]
@mattmasson, I'm not seeing this fixed in that release.

````
c:\Users\ben\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.1.3\.nuget\Microsoft.PowerQuery.SdkTools.2.110.2\tools\pqtest.exe run-test 
--extension c:\Users\ben\Desktop\TestConnector\BenTest\bin\AnyCPU\Debug\BenTest.mez 
--queryFile c:\Users\ben\Desktop\TestConnector\BenTest\BenTest.query.pq 
--prettyPrint

[3:47:59 PM]	[Info]	RunTestBattery result [
    {
        "ActivityId": "c3617078-30b6-441f-8ec6-28a8782655d1",
        "DataSourceAnalysis": [
            {
                "Kind": "DataSource",
                "FunctionName": "BenTest.Contents",
                "DataSource": {
                    "kind": "BenTest",
                    "path": "BenTest"
                },
                "DataSourceReference": {
                    "protocol": "x-datasource",
                    "address": {
                        "kind": "BenTest",
                        "path": "BenTest"
                    },
                    "authentication": null,
                    "query": null
                },
                "ModuleName": null,
                "ModuleVersion": null
            }
        ],
        "Details": "let\r\n    Result = BenTest.Contents()\r\nin\r\n    Result",
        "EndTime": "2022-09-26T19:47:59.5830061+00:00",
        "Method": "PQTest.RunTest",
        "Name": "BenTest.query.pq",
        "StartTime": "2022-09-26T19:47:58.4574819+00:00",
        "Output": [
            {
                "Column1": "2022-01-01T00:00:00"
            },
            {
                "Column1": "2022-01-01T00:00:00"
            }
        ],
        "RowCount": 2,
        "Status": "Passed",
        "TraceFileLocation": null,
        "Type": "PQTest.Expression"
    }
]
````

**Comment by bgribaudo** [comment]
Still not seeing this fixed as of extension version 0.1.5 (tool package version: 2.110.2):

````
c:\Users\ben\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.1.5\.nuget\Microsoft.PowerQuery.SdkTools.2.110.2\tools\PQTest.exe run-test
				--extension "c:\Users\ben\Desktop\TestConnector\BenTest\bin\AnyCPU\Debug\BenTest.mez"
				--queryFile "c:\Users\ben\Desktop\TestConnector\BenTest\BenTest.query.pq"
				--prettyPrint
[9:14:03 AM]	[Info]	RunTestBattery result [
    {
        "ActivityId": "3ae552ca-7820-4f18-9463-5ef9aff22ae3",
        "DataSourceAnalysis": [
            {
                "Kind": "DataSource",
                "FunctionName": "BenTest.Contents",
                "DataSource": {
                    "kind": "BenTest",
                    "path": "BenTest"
                },
                "DataSourceReference": {
                    "protocol": "x-datasource",
                    "address": {
                        "kind": "BenTest",
                        "path": "BenTest"
                    },
                    "authentication": null,
                    "query": null
                },
                "ModuleName": null,
                "ModuleVersion": null
            }
        ],
        "Details": "let\r\n    Result = BenTest.Contents()\r\nin\r\n    Result",
        "EndTime": "2022-09-28T13:14:03.4022784+00:00",
        "Method": "PQTest.RunTest",
        "Name": "BenTest.query.pq",
        "StartTime": "2022-09-28T13:14:02.2461339+00:00",
        "Output": [
            {
                "Column1": "2022-01-01T00:00:00"
            },
            {
                "Column1": "2022-01-01T00:00:00"
            }
        ],
        "RowCount": 2,
        "Status": "Passed",
        "TraceFileLocation": null,
        "Type": "PQTest.Expression"
    }
]

````

**Comment by bgribaudo** [comment]
@mattmasson / @ptyx507x , is there a way I can reopen this issue or should I create a new one?
